src/metabase/sync/fetch_metadata.clj

```diff
@@ -76,7 +76,7 @@
                                describe-fields-using-describe-table)]
       (cond->> (describe-fields-fn driver database args)
         ;; This is a workaround for the fact that [[mu/defn]] can't check reducible collections yet
-        (mu.fn/instrument-ns? *ns*)
+        (mu.fn/instrument-fn? *ns*)
```

Comment:
I renamed `mu.fn/instrument-ns?` to `mu.fn/instrument-fn?` to better reflect it's purpose, rather than how it works.

Reply:
`(instrument-fn? *ns*)` feels like a contradiction though?

To me `-ns?` still makes the most sense, but I could accept an argument for `-fns?`

cc @escherize

Reply:
I don't see the benefit of renaming this to `mu.fn/instrument-fn?` -- that reads like it should check a if a single function gets instrumented. 

Is it unclear as is? `mu.fn/instrument-functions-in-ns?` might capture your meaning better

Reply:
If it looks strange to you it probably looks strange to everyone else. I'll rename

Reply:
reverted back to `instrument-ns?` [6fd1868](https://github.com/metabase/metabase/pull/43942/commits/6fd1868ff827f6609c3120f29f9308e5b515865d)